src/components/Display.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Display = (props) => {
+  const { result } = props;
+
+  return (
+    <div className="Display">
+      <p>{result}</p>
+    </div>
+  );
+};
+
+Display.propTypes = {
+  result: PropTypes.string,
+};
+
+Display.defaultProps = {
+  result: 0,
```

Comment:
Since result is a string, make the default value a string as well.
```suggestion
Display.defaultProps = {
  result: '0',
```